.editorconfig

```diff
@@ -1,9 +1,7 @@
-root = true
-
-[*]
-end_of_line = lf
-insert_final_newline = true
-trim_trailing_whitespace = true
-charset = utf-8
-indent_style = space
-indent_size = 4
+root = true
+
+[*]
+end_of_line = lf
+indent_size = 2
+indent_style = space

```

Comment:
If we're changing this, we could change `indent_style` to `tab` and then setting the tab width is just up to each person via their editor settings. 

What do you think?

Reply:
i think given this is the same in other chai repos, we should probably do that across the board in a standalone PR